library/src/main/java/com/cottacush/android/currencyedittext/utils/Constants.kt

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2019 Cotta & Cush Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cottacush.android.currencyedittext.utils
+
+object Constants {
+
+    object Messages {
+        const val MAX_DECIMAL_DIGITS_ERROR = "Maximum number of Decimal Places must be a positive integer"
```

Comment:
I don't think we need to extract the error message to a constant. It's usually more readable to leave an exception message directly with the throw statement.